frontend/src/framework/theme/styles/components.styles.ts

```diff
@@ -3,6 +3,55 @@ import { colors } from './colors.styles.js';
 const components = {
     Heading: {
         baseStyle: {
+            color: 'text.white',
```

Comment:
I think it is better to change 'text.white' to simply 'white' because we have no definition for 'text.white' (only text.default and text.accent) but we have defined 'white' color in our colors customization.